HighAvailabilityModule.Storage.Server/startup.sql

```diff
@@ -38,9 +38,9 @@ IF OBJECT_ID('SetDataEntry') IS NOT NULL
 	DROP PROCEDURE SetDataEntry;
 GO
 CREATE PROCEDURE SetDataEntry
-	@dpath nvarchar(50),
-	@dkey nvarchar(50),
-	@dvalue nvarchar(50),
+	@dpath nvarchar(4000),
```

Comment:
Can we use [nvarchar(max)](https://stackoverflow.com/questions/11131958/what-is-the-maximum-characters-for-the-nvarcharmax/11131977) ?

Reply:
Primary keys cannot set to be `max` and the max length is 900.